### PR TITLE
LPS-170393 | Test Automation | Master

### PR DESCRIPTION
**Pull request by rafaellajordao**
# Add a Scope Selector to the Segment Editor

Hi! :)
This pr is corresponding to this ticket: 
- [LPS-168141](https://issues.liferay.com/browse/LPS-168141)
- [LPS-170393](https://issues.liferay.com/browse/LPS-170393)

| Testing Scenarios  | Test Strategy | Test File | Test Name |
| ------------- | ------------- | ------------- | ------------- |
| **Given** a site is already selected as site scope  **When** the user selects another site as scope  **Then** the new site should replace the previous one | Medium  | SegmentationCreateSegment | CanSelectSiteScope |
| **Given** a segment entry is scoped to a site **When** user navigates to another site **Then** the segment can not be seen  | High  | SegmentationCreateSegment | ViewSegmentScopedIndividualSiteIsNotDisplayed |
| **Given** a segment entry is scoped to a site **When** user navigates to that site **Then** the segment can be used | High | SegmentationCreateSegment | CanUseSegmentScopedIndividualSite |
| **Given** a segment entry is scoped to global site **When** user navigates to any site **Then** the segment can be used | Critical | SegmentationCreateSegment | ViewSegmentScopedGlobalDisplayedOnAnySite |
| **Given** the scope card at the segment editor **When** the user selects a site **And** saves the segment  **Then** the select button disappears | Medium | SegmentationCreateSegment | ViewSelectButtonIsNotPresentAfterSaveSegment|

### CODE REVIEW

**Comment by rafaellajordao** [comment]
ci:test:sf

**Comment by rafaellajordao** [comment]
ci:test:tango

**Comment by rafaellajordao** [comment]
ci:test:sf

**Comment by rafaellajordao** [comment]
ci:test:tango

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/segmentation/SegmentationCreateSegment.testcase [hunk 1]
```diff
@@ -2755,7 +2830,9 @@ definition {
 				locator1 = "Segmentation#CARD_HEADING",
 				panelHeader = "Scope");
 
-			AssertNotVisible(locator1 = "Segmentation#SCOPE_CARD_SITE");
+			AssertNotVisible(
```
Comment (CarlosBrichete) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/segmentation/SegmentationCreateSegment.testcase:2833]: Why did you add this change in this test?

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/segmentation/SegmentationCreateSegment.testcase [hunk 2]
```diff
@@ -3351,6 +3428,118 @@ definition {
...
+
+			Segmentation.configureScope(site = "Global");
+
+			Button.clickSave();
+
+			Alert.viewSuccessMessage();
+		}
+
+		task ("When user navigates to any site") {
+			Segmentation.openSegmentsAdmin(siteURLKey = "guest");
```
Comment (CarlosBrichete) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/segmentation/SegmentationCreateSegment.testcase:3454]: Here better use the Test Site Name instead of the guest site

**Review by CarlosBrichete — CHANGES_REQUESTED** [review]

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/segmentation/SegmentationCreateSegment.testcase (hunk 1 above)
Comment (rafaellajordao) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/segmentation/SegmentationCreateSegment.testcase:2833]: by the way the path was made it was finding the message above the site name and not the site name, I refactored the path and needed to correct the test that uses this path. 

But I forgot that the tests have the feature flag property! I will remove this commit but when the feature flag is removed this test will probably need a fix!

**Comment by rafaellajordao** [comment]
ci:test:sf

**Comment by rafaellajordao** [comment]
ci:test:tango

**Comment by rafaellajordao** [comment]
> ###  ci:test:tango - 194 out of 212 jobs passed in 3 hours 57 minutes
> Click here for more details.

these failures have nothing to do with the test automation :)

**Review by CarlosBrichete — APPROVED** [review]

**Comment by CarlosBrichete** [comment]
ci:forward
